rest/context.go

```diff
@@ -325,6 +348,64 @@ func (ctx *gorillaRequestContext) NextURL() (string, error) {
 	return u.String(), nil
 }
 
+type RouteVars map[string]string
```

Comment:
Please add a docstring explaining why this exists and for what it is used. It would be nice to have an example too, I've already forgotten the details of what this looks like :)